src/lib/vuln.ts

```diff
@@ -27,11 +27,26 @@ export function getUpgrades(
     const actionableRemediation: ActionableRemediation = {
       upgradeFrom: key,
       upgradeTo,
-      severityScore: getSeverityScore(vulns),
       vulns,
+      severityScore: getSeverityScore(vulns),
     };
     result.push(actionableRemediation);
   });
   const sortedResult = _.orderBy(result, 'severityScore', 'desc');
   return sortedResult;
 }
+
+export function addIssueDataToPatch(remediation, vulnerabilities) {
```

Comment:
is getActionablePatches a better function name?
type inputs and outputs, even if it's just `any` and `ActionablePatch[]`

Reply:
renamed